api/middleware/authProfile.js

```diff
@@ -0,0 +1,37 @@
+const createError = require('http-errors');
+const Profiles = require('../profile/profileModel');
+// const jwt = require('express-jwt');
```

Comment:
Do we need this commented-out code? if not, lets remove it.

Reply:
removed it!